### PR TITLE
refactor(schemas): global, chat message and chat mentions

### PR DESCRIPTION
**Pull request by Defferrard**

### CODE REVIEW

File: src/schemas/global.ts [hunk 1]
```diff
@@ -1,57 +1,69 @@
...
-        name: { type: 'string' },
-        code: { type: 'string' },
-        message: { type: 'string' },
-        statusCode: { type: 'number' },
-        data: {},
-        origin: { type: 'string' },
-        // stack: {}
-      },
+  ),
+);
```
Comment (pyphilia) [inline src/schemas/global.ts:42]: Do you think it is necessary?

File: src/services/item/plugins/app/appAction/schemas.ts [hunk 2]
```diff
@@ -1,4 +1,6 @@
+import { customType } from '../../../../../plugins/typebox';
 import { accountSchemaRef } from '../../../../account/schemas';
+import { itemIdSchemaRef } from '../../itemLike/schemas';
```
Comment (pyphilia) [inline src/services/item/plugins/app/appAction/schemas.ts:3]: Should this come from `/item/schemas` instead?

**Review by pyphilia — APPROVED** [review]
LGTM!

File: src/schemas/global.ts (hunk 1 above)
Comment (Defferrard) [reply src/schemas/global.ts:42]: I don't think it is necessary, I can remove it for sure !

File: src/services/item/plugins/app/appAction/schemas.ts (hunk 2 above)
Comment (Defferrard) [reply src/services/item/plugins/app/appAction/schemas.ts:3]: I'll move it on the refactor of Items Schemas 👍🏻

File: src/services/item/plugins/app/appAction/schemas.ts [hunk 3]
```diff
@@ -37,13 +39,13 @@ const create = {
 };
 
 const getForOne = {
-  params: { $ref: 'https://graasp.org/apps/#/definitions/itemIdParam' },
+  params: itemIdSchemaRef,
   querystring: {
     type: 'object',
     properties: {
       /** @deprecated use accountId */
       memberId: { $ref: 'https://graasp.org/#/definitions/uuid', deprecated: true },
```
Comment (ReidyT) [inline src/services/item/plugins/app/appAction/schemas.ts:47]:
```suggestion
      memberId: customType.UUID({ deprecated: true }),
```

File: src/services/member/plugins/thumbnail/schemas.ts [hunk 4]
```diff
@@ -34,15 +36,9 @@ const download = {
...
-    200: {
-      type: 'string',
-    },
-    '4xx': {
-      $ref: 'https://graasp.org/#/definitions/error',
-    },
-    500: {
-      $ref: 'https://graasp.org/#/definitions/error',
-    },
+    200: errorSchemaRef,
```
Comment (ReidyT) [inline src/services/member/plugins/thumbnail/schemas.ts:39]: Use the enum here?

File: src/services/member/plugins/thumbnail/schemas.ts [hunk 5]
```diff
@@ -34,15 +36,9 @@ const download = {
...
+    '4xx': errorSchemaRef,
+    500: errorSchemaRef,
```
Comment (ReidyT) [inline src/services/member/plugins/thumbnail/schemas.ts:41]: Use the enum here too?

**Review by ReidyT — APPROVED** [review]
LGTM, I just let you some comments 🫠

**Pull request merged**